specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2024-05-15/managedCassandra.json

```diff
@@ -0,0 +1,1426 @@
+                      "type": "number",
+                      "format": "double"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "ManagedCassandraReaperStatus": {
+      "type": "object",
+      "properties": {
+        "healthy": {
+          "type": "boolean"
+        },
+        "repairRunIds": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "repairSchedules": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "ConnectionError": {
+      "type": "object",
+      "properties": {
+        "connectionState": {
+          "description": "The kind of connection error that occurred.",
+          "type": "string",
+          "enum": [
+            "Unknown",
+            "OK",
+            "OperatorToDataCenterNetworkError",
+            "DatacenterToDatacenterNetworkError",
+            "InternalOperatorToDataCenterCertificateError",
+            "InternalError"
+          ],
+          "x-ms-enum": {
+            "name": "ConnectionState",
+            "modelAsString": true
+          }
+        },
+        "iPFrom": {
+          "description": "The IP of host that originated the failed connection.",
+          "type": "string"
+        },
+        "iPTo": {
+          "description": "The IP that the connection attempted to reach.",
+          "type": "string"
+        },
+        "port": {
+          "description": "The TCP port the connection was attempted on.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "exception": {
+          "description": "Detailed error message about the failed connection.",
+          "type": "string"
+        }
+      }
+    },
+    "CassandraError": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The code of error that occurred.",
+          "type": "string"
+        },
+        "message": {
+          "description": "The message of the error.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The target resource of the error.",
+          "type": "string"
+        },
+        "additionalErrorInfo": {
+          "description": "Additional information about the error.",
+          "type": "string"
+        }
+      }
+    },
+    "CommandPostBody": {
+      "description": "Specification of which command to run where",
+      "type": "object",
+      "required": [
+        "command",
+        "host"
+      ],
+      "properties": {
+        "command": {
+          "type": "string",
+          "description": "The command which should be run"
+        },
+        "arguments": {
+          "type": "object",
+          "description": "The arguments for the command to be run",
+          "additionalProperties": {
```

Comment:
Could you add a suppression on this, if this is going to make a breaking change updating this ?

Reply:
This piece of code is not updated in this PR.